### DISCUSSION TITLE
Zod `object` naming

### DISCUSSION THREAD

**Discussion by hugetim** [anvilistas/anvil-labs#114] [General]
Could we call the `object` type "dict" instead, following the Python terminology rather than the JS?

I guess the disadvantage would be more distance from the original zod.dev docs.

(The zod module is a dream come true by the way. Thank you! 🎉🎁🚀)

**Comment by s-cork** [comment]
It actually supports parsing any dict-like object so `z.dict` wouldn't be totally accurate.

I think technically you could parse a table row with an object schema. (Though I've not tried yet. And only with accelerated tables turned on)

`z.mapping` might be better since that's also used in python for dict-like objects. 
I'm happy to just alias it. 

Thoughts?

**Reply by hugetim** [reply]
Yeah, `mapping` sounds good.

**Reply by s-cork** [reply]
done in #115
